### PR TITLE
Prepare 9.2.0 release

### PR DESCRIPTION
**Pull request by michaelwoerister**
Makes a couple of methods in `ProfilingData` public in order to make it easier to start supporting older file formats in v10.0.0.

r? @rylev & @wesleywiser

### CODE REVIEW

**Review by rylev — APPROVED** [review]
Looks great! Technically I would say we should update the README to point the install to this new version, but if we're going to release v10.0.0 right after this, it's not a blocker.

File: analyzeme/src/profiling_data.rs [hunk 1]
```diff
@@ -174,6 +174,19 @@ impl ProfilingData {
             thread_id: raw_event.thread_id,
         }
     }
+
+    // This method is added only in order to make it easier to fix
+    // https://github.com/rust-lang/measureme/issues/173 in the future.
+    //
+    // It is marked as "internal" because we want to avoid that downstream tooling
+    // depends on RawEvent directly. Do not use this method directly.
```
Comment (rylev) [inline analyzeme/src/profiling_data.rs:182]: Hmm this makes me a bit worried. If we don't strictly needed it right perhaps we should just not include it in this release?

**Review by wesleywiser — APPROVED** [review]
We should also update the `CHANGELOG.md` for this release.

File: analyzeme/src/profiling_data.rs (hunk 1 above)
Comment (michaelwoerister) [reply analyzeme/src/profiling_data.rs:182]: I don't have a strong opinion about this. I can remove it if you prefer.

**Comment by michaelwoerister** [comment]
> We should also update the CHANGELOG.md for this release.

Done.

File: analyzeme/src/profiling_data.rs (hunk 1 above)
Comment (michaelwoerister) [reply analyzeme/src/profiling_data.rs]: Removed

**Pull request merged**